### ISSUE TITLE
Questions for p_gens

### ISSUE THREAD

**Issue by HanGuo97** [abisee/pointer-generator#56]
Thanks for this awesome implementation!

I have a question regarding the dimension of `p_gens`. Why is it a list (with length equals the number of decoder steps) of scalars, but not a list of `[batch_size]`?

**Comment by abisee** [comment]
You're right, the comments were wrong. Now [fixed](https://github.com/abisee/pointer-generator/commit/0cdcaeeaf8f42d4d64ec2ed09eb2f0158cd0db8f).
